pkg/unit_test.go

```diff
@@ -0,0 +1,155 @@
+package openapiart_test
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"reflect"
+	"testing"
+
+	. "github.com/open-traffic-generator/openapiart/pkg"
```

Comment:
. "github.com/open-traffic-generator/openapiart/pkg"

instead of ".", use "openapiart". this will require prepending some structs/funcs with openapiart.

Reply:
updated